recipes/ut/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+
+
+class UTConan(ConanFile):
+    name = "ut"
+    description = "C++20 single header/single module, "
+    "macro-free micro Unit Testing Framework"
+    topics = ("conan", "UT", "header-only", "unit-test", "tdd", "bdd")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://boost-ext.github.io/ut/"
+    license = "Boost"
+    settings = "os", "compiler", "arch", "build_type"
+    no_copy_source = True
+
+    def configure(self):
+        self._validate_compiler_settings()
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("ut-" + self.version, self._source_subfolder)
+        tools.download("https://www.boost.org/LICENSE_1_0.txt", "LICENSE",
+                       sha256="c9bff75738922193e67fa726fa225535870d2aa1059f914"
+                       "52c411736284ad566")
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses")
+        self.copy(os.path.join("include", "boost", "ut.hpp"),
+                  src=self._source_subfolder)
+
+    def package_id(self):
+        self.info.header_only()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "UT"
+        self.cpp_info.names["cmake_find_package_multi"] = "UT"
```

Comment:
This will no give the effect you are looking for... I would encourage you to add the `find_package(ut)` and `target_link_libs(... boost::ut)` to ensure that it works. 

You'll need to use the Conan "components" features https://github.com/conan-io/conan-center-index/tree/master/recipes/cpprestsdk/all/test_package for an example

Reply:
I think I've got that figured out.
Thanks for the link! I'm not sure if I need to avoid exporting the `boost::boost` CMake target in case of potential conflicts with Boost.
If I should, just let me know how, and I'll make the change.